### PR TITLE
Fixes golint in test/utils/image

### PR DESCRIPTION
**Pull request by jonfriesen**
**What this PR does / why we need it**:
This change fixes the test/utils/image package golint errors.

**Which issue(s) this PR fixes**:
refer #68026

**Special notes for your reviewer**:
I renamed `ImageConfig` to `Config` to fix stuttering (eg. `image.ImageConfig`). I couldn't find a spot where this would break other packages. Though I'm not exactly sure what the best practice is on renaming a public struct. Guidance would be appreciated  :) 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: test/utils/image/manifest.go [hunk 1]
```diff
@@ -79,55 +85,63 @@ var (
...
+	Netexec                  = Config{e2eRegistry, "netexec", "1.0"}
+	Nettest                  = Config{e2eRegistry, "nettest", "1.0"}
+	Nginx                    = Config{dockerLibraryRegistry, "nginx", "1.14-alpine"}
+	NginxNew                 = Config{dockerLibraryRegistry, "nginx", "1.15-alpine"}
+	Nonewprivs               = Config{e2eRegistry, "nonewprivs", "1.0"}
+	NoSnatTest               = Config{e2eRegistry, "no-snat-test", "1.0"}
+	NoSnatTestProxy          = Config{e2eRegistry, "no-snat-test-proxy", "1.0"}
+)
+
+// When these values are updated, also update cmd/kubelet/app/options/container_runtime.go
```
[the remark this answers is not in the document]
Comment (jonfriesen) [reply test/utils/image/manifest.go]: ah thanks, that wasn't clear on that. updated the comment to be pause specific

**Comment by ixdy** [comment]
/approve

Looks like this needs rebase, though.

**Comment by jonfriesen** [comment]
oh... yeah I think I messed up this PR

**Pull request closed without merging**